File: app/src/main/java/com/example/watering/investrecord/DBHelper/AccountDBHelper.java
package com.example.watering.investrecord.DBHelper;

import android.content.Context;

/**
 * Created by watering on 17. 10. 21.
 */

@SuppressWarnings("DefaultFileTemplate")
public class AccountDBHelper extends DBHelper {

    public AccountDBHelper(Context context) {
        super(context);

        TABLE_NAME = "tbl_Account";
        COLUMNS = new String [] {"institute TEXT","number TEXT UNIQUE","description TEXT",
                "id_group INTEGER"};
    }
}

File: app/src/main/java/com/example/watering/investrecord/dbhelper/AccountDBHelper.kt
package com.example.watering.investrecord.dbhelper

import android.content.Context

/**
 * Created by watering on 17. 10. 21.
 */

class AccountDBHelper(context: Context) : DBHelper(context) {

    init {
        TABLE_NAME = "tbl_Account"
        COLUMNS = arrayOf("institute TEXT", "number TEXT UNIQUE", "description TEXT", "id_group INTEGER")
    }
}
